dor/entrypoints/api/catalog.py

```diff
@@ -0,0 +1,58 @@
+import os
+import uuid
+
+from dor.service_layer import catalog_service
+import sqlalchemy
+from fastapi import APIRouter, Depends, status
+from fastapi.responses import JSONResponse
+
+from dor.adapters.catalog import SqlalchemyCatalog
+
+
+catalog_router = APIRouter(prefix="/catalog")
+
+url = sqlalchemy.engine.URL.create(
+    drivername="postgresql+psycopg",
+    username=os.environ["POSTGRES_USER"],
+    password=os.environ["POSTGRES_PASSWORD"],
+    host=os.environ["POSTGRES_HOST"],
+    database="dor_test"
+)
+ENGINE = sqlalchemy.create_engine(url)
+
+
+def get_db_session():
+    with sqlalchemy.orm.Session(ENGINE) as session:
+        yield session
+
+
```

Comment:
What do you think about adding `http://localhost:8000/api/v1/catalog/bins` as a route that simply returns the number of bins in the catalog?

Reply:
Maybe `stats`?

Reply:
Interesting idea, what else besides a count of bins would we provide? I'd suggest we tackle this as a separate issue or story.

Reply:
I’ll go along with `stats` and a separate issue. It is just with an empty database there aren’t any paths that return an okay.

Reply:
Yeah, that's why we need to add the catalog step in the workflow!